docs/api.rst

```diff
@@ -6,7 +6,38 @@
 
 
 LIBBPF API
-==================
+==========
```

Comment:
Please remind me, what did we agree about with where libbpf docs live? Not in linux repo anymore, right? I see that we have libbpf_build.rst, libbpf_naming_convention.rst, index.rst that are copied from Linux repo. But api.rst is only in libbpf/libbpf on Github? If we moved everything to github, why didn't we move existing docs here as well?

Reply:
So as it stands now the libbpf documentation (the files you listed) live in the kernel repo, except for the API documentation (specifically `api.rst`) which only lives in the mirror. This is because the API documentation should be versioned and having the 'latest' API docs in the kernel repo, and therefore on kernel.org/doc is ambiguous and confusing.

The configuration files for sphinx/doxygen/readthedocs (what powers libbpf.readthedocs.org) also live in the mirror.

Reply:
That's why index.rst links to libbpf.readthedocs.org explicitly, not a relative link.

Reply:
something is wrong with rendering, btw 
<img width="773" alt="Screen Shot 2021-12-22 at 1 42 30 PM" src="https://user-images.githubusercontent.com/508075/147158896-6d417154-ae65-494e-9ad7-3e406346ca20.png">

Reply:
Should be fixed now.